docs/installpackage.md

```diff
@@ -0,0 +1,59 @@
+# Installing Our Package
+
+Our package name is currently `EducationExtension`.
+
+Our top level namespace is currently `Extension`.
+
+Our GPR source is `https://nuget.pkg.github.com/dotnet-interactive-learning/index.json`.
+
+## From GitHub Package Registry (GPR) NuGet Source
+
+1. Install NuGet CLI. See [Nuget CLI Reference](https://docs.microsoft.com/en-us/nuget/reference/nuget-exe-cli-reference). 
```

Comment:
Is there a way to do this with the `dotnet` CLI? It's a lot more commonly used and will already be present for people who have the .NET SDK installed.